src/ui/components/map/providers/mapboxmapprovider.js

```diff
@@ -3,45 +3,71 @@
 import MapProvider from './mapprovider';
 import DOM from '../../../dom/dom';
 
-/* global mapboxgl */
+/* global mapboxgl, MapboxLanguage */
 
 /**
  * MapBoxMapProvider is an implementation of a MapProvider
  * that handles the integration with the third party API to expose maps.
  * @extends MapProvider
  */
 export default class MapBoxMapProvider extends MapProvider {
+  constructor (opts = {}, systemOpts = {}) {
+    super(opts, systemOpts);
+
+    this._isMapboxLoaded = false;
```

Comment:
What's the difference between `_isMapboxLoaded` and `isLoaded`?

Reply:
The idea was to set `isLoaded` to true once both `_isMapboxLoaded` and `_isMapboxScriptLoaded` were true. However this is now outdated